test/metabase/lib/segment_test.cljc

```diff
@@ -6,18 +6,19 @@
    [metabase.lib.metadata :as lib.metadata]
    [metabase.lib.metadata.calculation :as lib.metadata.calculation]
    [metabase.lib.test-metadata :as meta]
-   [metabase.lib.test-util :as lib.tu]))
+   [metabase.lib.test-util :as lib.tu]
+   [metabase.lib.test-util.macros :as lib.tu.macros]))
 
 #?(:cljs (comment metabase.test-runner.assert-exprs.approximately-equal/keep-me))
 
 (def ^:private segment-id 100)
 
 (def ^:private segment-definition
-  {:source-table (meta/id :venues)
-   :aggregation  [[:count]]
```

Comment:
This definition is unexpected. In the FE, I see no way for adding aggregations. That's what distinguished  segments from legacy metrics, AFAIK.

Reply:
See above; they do seem to have aggregations.